frontend/src/pages/project/main/ProjectItem.module.scss

```diff
@@ -0,0 +1,27 @@
+@import "styles/spacings";
+@import "styles/colors";
+@import "styles/typography";
+
+.item {
+  display: flex;
+  flex-direction: column;
+
+  .main {
```

Comment:
Maybe it would make more sense to name this  `header` and also change the `timestamp` class name to `footer` for example?

I think in many cases when some UI element has multiple (2 or 3) horizontal sections, it could make sense to view it from the "header" / "body" / "footer" UI model.